pkg/graph/resolvers/resolver_test_utilities.go

```diff
@@ -49,18 +52,17 @@ func GetDefaultTestConfigs() *TestConfigs {
 	return &tc
 }
 
-func createS3Client() s3.S3Client {
-	config := testhelpers.NewConfig()
+func createS3Client(viperConfig *viper.Viper) s3.S3Client {
 
 	s3Cfg := s3.Config{
-		Bucket:  config.GetString(appconfig.AWSS3FileUploadBucket),
-		Region:  config.GetString(appconfig.AWSRegion),
+		Bucket:  viperConfig.GetString(appconfig.AWSS3FileUploadBucket),
+		Region:  viperConfig.GetString(appconfig.AWSRegion),
 		IsLocal: true,
 	}
 	//OS ENV won't get environment variables set by VSCODE for debugging
-	_ = os.Setenv(appconfig.LocalMinioAddressKey, config.GetString(appconfig.LocalMinioAddressKey))
-	_ = os.Setenv(appconfig.LocalMinioS3AccessKey, config.GetString(appconfig.LocalMinioS3AccessKey))
-	_ = os.Setenv(appconfig.LocalMinioS3SecretKey, config.GetString(appconfig.LocalMinioS3SecretKey))
+	_ = os.Setenv(appconfig.LocalMinioAddressKey, viperConfig.GetString(appconfig.LocalMinioAddressKey))
+	_ = os.Setenv(appconfig.LocalMinioS3AccessKey, viperConfig.GetString(appconfig.LocalMinioS3AccessKey))
+	_ = os.Setenv(appconfig.LocalMinioS3SecretKey, viperConfig.GetString(appconfig.LocalMinioS3SecretKey))
```

Comment:
Can you clarify these lines? I know they're not entirely new in this PR, but I'm curious why they're actually needed. The only time we should be reading env vars is at the _very_ start of the program when we call `Viper.AutomaticEnv()` -- after that actual env vars shouldn't be needed (since everything's embedded in the viper config)

Reply:
Thanks Clay, it's been a while, but from what I'm remembering and the comment, this enabled debugging through the vscode testing tab. newS3 client calls os.Getenv instead of getting environment variables from viperConfig. This was a work around to get the variables so you can still debug and run tests with VSCode.

This is also part of the entrypoint for the test files (it's not called except for testing).

Reply:
I think the other possibility is to rework NewS3Client to also take a viper.Viper param, and use that to get the envVar. I suspect it was originally done this way just for simplicity